Scripts/powershell/common.psm1

```diff
@@ -2,19 +2,20 @@
 function Invoke-CoyoteTool([String]$cmd, [String]$dotnet, [String]$framework, [String]$target, [String]$key) {
     Write-Comment -prefix "..." -text "Rewriting '$target' ($framework)" -color "white"
 
-    $tool = "./bin/$framework/coyote.exe"
+    $tool = Join-Path -Path "." -ChildPath "bin" -AdditionalChildPath @($framework, "coyote.exe")
     $command = "$cmd $target"
 
     if (-not (Test-Path $tool)) {
         $tool = $dotnet
-        $command = "./bin/$framework/coyote.dll $cmd $target"
+        $coyote = Join-Path -Path "." -ChildPath "bin" -AdditionalChildPath @($framework, "coyote.dll")
+        $command = "$coyote $cmd $target"
     }
 
-    if ($command -eq "rewrite" -and $framework -ne "netcoreapp3.1" -and $framework -ne "net5.0" -and [System.Environment]::OSVersion.Platform -eq "Win32NT") {
+    if ($command -eq "rewrite" -and $framework -ne "netcoreapp3.1" -and $framework -ne "net5.0" -and $IsWindows) {
```

Comment:
I didn't know this existed, that's pretty cool! Seems these shipped with one of the latest PowerShell versions :)

Reply:
yep I found it when i was looking for another powershell feature